client/rpc-api/src/state/mod.rs

```diff
@@ -136,4 +136,98 @@ pub trait StateApi<Hash> {
 	fn unsubscribe_storage(
 		&self, metadata: Option<Self::Metadata>, id: SubscriptionId
 	) -> RpcResult<bool>;
+
+	/// The `state_traceBlock` RPC provides a way to trace the re-execution of a single
+	/// block, collecting Spans and Events from both the client and the relevant WASM runtime.
+	/// The Spans and Events are conceptually equivalent to those from the [Tracing][1] crate.
+	///
+	/// The structure of the traces follows that of the block execution pipeline, so meaningful
+	/// interpretation of the traces requires an understanding of the Substrate chain's block
+	/// execution.
+	///
+	/// [Link to conceptual map of trace structure for Polkadot and Kusama block execution.][2]
+	///
+	/// [1]: https://crates.io/crates/tracing
+	/// [2]:
+	/// https://github.com/paritytech/substrate/tree/master/client/rpc-api/TraceBlock_Trace_Hierarchy.png
```

Comment:
I think this link should work once the PR is merged

Reply:
Not sure why it won't show the PNG in this diff, but for now the link is here: 
 https://docs.google.com/drawings/d/1vZoJo9jaXlz0LmrdTOgHck9_1LsfuQPRmTr-5g1tOis/edit?usp=sharing